allen-cell-animated/simularium-website
# Feature/viewer controller refactor

Problem
=======
Update usage of SimulariumController in simularium-viewer 2.8.0

Solution
========
Update viewer version and modify calls to `changeFile`

## Type of change


Change summary:
---------------
Some reformatting happened due to format-on-save in my editor.  If you view the side by side diff, you'll see the only changes are to the simularium-viewer version and the calls to `changeFile`

Steps to Verify:
----------------
run website and verify that trajectories load, drag and drop works, and state isn't corrupted when the trajectory is changed.
